# Exercise 15.1b solution

For a circular cross-section with diameter $d$:
- $A_i = \pi (d/2)^2 = \pi d^2 / 4$
- $p = \pi d$

Substituting into the space constant expression:

$$
\lambda_{\mathrm{m}} = \sqrt{\frac{R_{\mathrm{m}} A_i}{p \cdot R_c}} = \sqrt{\frac{R_{\mathrm{m}} \cdot \pi d^2/4}{\pi d \cdot R_c}} = \sqrt{\frac{R_{\mathrm{m}} \cdot d}{4 R_c}}
$$

In [1]:
import numpy as np

# Parameters
R_m = 7000  # Ohm cm^2
R_c = 150  # Ohm cm
d = 10e-4  # cm (10 µm = 10e-4 cm)

# Cross-section and circumference
A_i = np.pi * (d / 2) ** 2
p = np.pi * d

# Space constant (assuming r_e << r_i)
lambda_m = np.sqrt(R_m * A_i / (p * R_c))

print(f"Space constant: {lambda_m:.4f} cm")
print(f"Space constant: {lambda_m * 10:.2f} mm")

Space constant: 0.1080 cm
Space constant: 1.08 mm


A human motor neuron axon can be up to ~100 cm long. The space constant is only

In [2]:
print(f"{lambda_m:.4f} cm ({lambda_m * 10:.2f} mm).")

0.1080 cm (1.08 mm).


That is roughly

In [3]:
print(f"{100 / lambda_m:.0f} space constants per metre.")

926 space constants per metre.


After just one space constant, the signal has decayed to ~37%. After 10 space constants, it is essentially gone ($\exp(-926) \approx 0$).

Passive conduction alone cannot carry a signal over such distances. Active, regenerating conduction (action potentials) is essential.